src/styles/blocks/catalog.scss

```
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transition-duration;
```

```suggestion
    transition-duration: transform $transition-duration, color $transition-duration;
```

You also need to specify the properties you change on hover